src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -645,8 +645,6 @@ export class TerminalInstance extends Disposable implements ITerminalInstance {
 
 		this._attachBarrier.open();
 
-		this.xterm?.attachToElement(container);
```

Comment:
This was added in https://github.com/microsoft/vscode/commit/e667055f4f8fe8ca77d653b6cc11ff6d5724b9d1
cc @Tyriar